config/prow/plugins.yaml

```diff
@@ -1,6 +1,11 @@
 # Plugin configuration
 ---
 triggers:
+- repos:
+  - kubernetes-sigs/cluster-api
+  join_org_url: "https://git.k8s.io/community/community-membership.md#member"
+  only_org_members: true
+  trigger_github_workflows: true
```

Comment:
I think I have to put this first in the array because the trigger plugin only takes the first config entry which matches the org or org/repo: https://github.com/kubernetes/test-infra/blob/140be8ec08637c68a99857b08c534a4fb737681e/prow/plugins/config.go#L858-L870